packages/generators/templates/component/{{name}}.{{mdx}}

```diff
@@ -11,10 +11,6 @@ import { {{name}} } from ".";
 
 # {{ titleCase name }}
 
-<ComponentStatus stage="pre" responsive="no" accessible="no" />
```

Comment:
If we are removing this, we can remove the `import { ComponentStatus } from "@jobber/docx";` from line `9`